### ISSUE TITLE
LLAMA 2 support [Question] [Enhancement]

### ISSUE THREAD

**Issue by trholding** [OpenNMT/CTranslate2#1355]
Since LLAMA is supported (haven't tried yet.) I am wondering if this would support LLAMA 2. 

I have been using CTranslate2 only for whisper and nllb translation.

**Comment by Apoorv7092** [comment]
Yes it does support LLAMA 2. Run the conversion code same as for LLAMA(I tried with only 7b model, so only can vouch for that)

**Comment by hobodrifterdavid** [comment]
Check here: https://github.com/OpenNMT/CTranslate2/issues/1351
7B and 13B are working fine, 70B may need further work.

**Comment by guillaumekln** [comment]
The Llama 2 architecture is already supported by the runtime (including the 70B version using GQA), but the Transformers converter required small changes to make the conversion run without errors (cf. #1354).

This change is included in version 3.17.1 which is currently being released (the build takes ~1h).

I prepared a small example that implements an interactive chat session with Llama 2:

https://github.com/OpenNMT/CTranslate2/tree/master/examples/llama2

**Comment by trholding** [comment]
@Apoorv7092 Thanks, at the time I tried it, it errored out on my ARM server. I used this model: https://huggingface.co/michaelfeil/ct2fast-Llama-2-7b-chat-hf and the example associated with it. I changed device type to cpu and type to int8.

@hobodrifterdavid Thank you, I'll test it and get back on that thread/issue

@guillaumekln thanks a ton. Will test it out!
